wls-gui-wahllokalsystem/src/components/common/inputs/BaseNumberInput.vue

```diff
@@ -1,19 +1,17 @@
 <template>
-  <v-text-field
-    ref="inputRef"
-    v-model="formattedValue"
+  <v-number-input
+    v-model="modelValue"
     :label="props.label"
-    clearable
-    variant="solo"
     :rules="props.rules"
-    type="number"
-    hide-spin-buttons
+    clearable
+    persistent-clear
```

Comment:
Alle anderen Eingabefelder zeigen ebenfalls nicht permanent das clearable an

```suggestion
    clearable
```

Reply:
ich bin dafür, dass wir es trotzdem drin lassen und ggf bei den anderen nachziehen. finde es ist sehr viel nutzerfreundlicher mit `persistant-clear`

Reply:
Ich hätte gedacht wir haben das mal mit Stefan besprochen. Allerdings konnte ich in den Aufzeichnungen dazu nichts finden.
Kannst du Stefan fragen wie es ihm lieber ist.

Reply:
seine antwort: 
> Ja, es ist besser, wenn das x immer zu sehen ist.

 ich mache für alle anderen inputs ein issue auf: #1876